pkg/usage/usage.go

```diff
@@ -0,0 +1,267 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package usage
+
+import (
+	node "github.com/openebs/cstor-csi/pkg/kubernetes/node"
+)
+
+// Usage struct represents all information about a usage metric sent to
+// Google Analytics with respect to the application
+type Usage struct {
+	// Embedded Event struct as we are currently only sending hits of type
+	// 'event'
+	Event
+
+	// https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#an
+	// use-case: cstor or jiva volume, or m-apiserver application
```

Comment:
n_p: m-apiserver

Reply:
changed to --> `use-case: cstor volume, cstor-operatos application`